src/length-value.js

```
@@ -73,6 +85,12 @@
     throw new TypeError('Not implemented yet');
   };
 
+  LengthValue.prototype._convertToCalcLength = function() {
```

This should probably throw an error saying it shouldn't be reached.